libvncclient/sockets.c

```diff
@@ -30,13 +30,16 @@
 # define _POSIX_SOURCE
 #endif
 #endif
+#include <rfb/rfbclient.h>
```

Comment:
Yes, we need `rfbconfig.h` in order to have `LIBVNCSERVER_HAVE_*`. Since `rfbclient.h` includes `rfbconfig.h` we can simply move it up. Alternatively an explicit `#include "rfbconfig.h"` could be added at the top.

Reply:
Let's do the latter please.